code/processes/rdb.q

```diff
@@ -184,13 +188,8 @@ reload:.rdb.reload
 /-subscribe to the tickerplant
 .rdb.subscribe[]
 
-/-check if the tickerplant has connected, block the process until a connection is established
-while[.rdb.notpconnected[];
-	/-while no connected make the process sleep for X seconds and then run the subscribe function again
-	.os.sleep[.rdb.tpconnsleepintv];
-	/-run the servers startup code again (to make connection to discovery)
-	.servers.startup[];
-	.rdb.subscribe[]]
+//check if tickerplant is available and if not exit with error 
+.servers.startupdependent[.rdb.requiredprocs;.rdb.tpconnsleepintv;5;.rdb.subscribe]
```

Comment:
think that `5` should perhaps be configurable in the same way as things like sleepintv etc.